source/extensions/filters/common/rbac/utility.h

```diff
@@ -34,8 +34,10 @@ struct RoleBasedAccessControlFilterStats {
   SHADOW_RBAC_FILTER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
-RoleBasedAccessControlFilterStats
-generateStats(const std::string& prefix, const std::string& shadow_prefix, Stats::Scope& scope);
+RoleBasedAccessControlFilterStats generateStats(const std::string& prefix,
```

Comment:
I would prefer to use absl::string_view here since you are changing this API anyway. Will it be too much trouble to change it?

Reply:
While I'd love to help out here, I'd prefer to merge this PR in its current form. I don't have the time right now to make this change and I'd rather avoid more conflicts, especially since https://github.com/envoyproxy/envoy/pull/32313 is related. Would it be ok to merge it as is?